lib/core/src/Cardano/Wallet/TokenMetadata.hs

```diff
@@ -356,6 +355,9 @@ data TokenMetadataError
         -- ^ Error from aeson decoding of JSON
     deriving (Generic, Show, Eq)
 
+instance NFData TokenMetadataError where
+    rnf = rnf . show
```

Comment:
Is this `show` right? 🤔

Reply:
Yes it's a cheap trick to make `NFData` instances, where some of the fields don't have `NFData`.
In this case the problem was `SomeException`.
`show` usually works well enough for the purpose of deeply evaluating some data.